src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+:root {
+  --color-text: #3d4e61;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
```

Comment:
it is redundant to reset padding for each selector. better reset only for elements which really need it